backend/services/session_manager.py

```
@@ -0,0 +1,595 @@
+            print(f"Error getting session statistics: {e}")
+            return {}
+        finally:
+            db.close()
+    
+    def start_cleanup_task(self):
+        """Start the background cleanup task"""
+        try:
+            loop = asyncio.get_event_loop()
+            if loop.is_running():
+                # If we're already in an event loop, create the task
+                asyncio.create_task(cleanup_task())
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)